yt_dlp/options.py

```diff
@@ -408,6 +408,10 @@ def _alias_callback(option, opt_str, value, parser, opts, nargs):
         help=(
             'Location of the main configuration file; either the path to the config or its containing directory '
             '("-" for stdin). Can be used multiple times and inside other configuration files'))
+    general.add_option(
+        '--plugin-locations',
+        dest='plugin_locations', metavar='PATH', action='append',
+        help='Location to search for plugin packages. Can be used multiple times to add multiple directories.')
```

Comment:
Similarly, here we should clarify that the PATH **needs** to be a directory

But more importantly: until the globals proposal is implemented, this option will only work with **extractor** plugins, and **not** postprocessor plugins (they are loaded too early). This needs to be documented in the help string for now

```suggestion
        help='Path to a directory for yt-dlp to search for plugins. Note that this currently only works for extractor plugins; postprocessor plugins can only be loaded from the default plugin directories. This option can be be used multiple times to add multiple directories')
```